samples/Shell/ash-env

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env ash
+echo "ash"
```

Comment:
We prefer to have samples from "real" applications (under permissive license). Alternatively, you can move those to `tests/fixtures/`.